.github/workflows/test.yml

```diff
@@ -183,10 +183,8 @@ jobs:
       run: |
         docker load --input image-master.tar
 
-    - name: Check out code into the Go module directory - from master
+    - name: Check out code into the Go module directory - from PR branch
```

Comment:
seems reasonable to me. We dont care about testing previous kind.sh or other executables from the repo, the point of the upgrade test is to exercise the difference between a HEAD built ovn-kube image. We can deploy with the latest kind.sh, tests, etc from the PR branch.